sample_data/vietnam_by_province_since_1996/vietnam_population_2019.csv

```diff
@@ -0,0 +1,64 @@
+NAME_1,Cartogram Data (eg. Population),Color,Inset,Label
+An Giang,8053663,,,
+Bà Rịa - Vũng Tàu,854679,,,
+Bắc Giang,530341,,,
+Bắc Kạn,313905,,,
+Bạc Liêu,784811,,,
+Bắc Ninh,730420,,,
+Bến Tre,598856,,,
+Bình Định,460196,,,
+Bình Dương,1248415,,,
+Bình Phước,821030,,,
+Bình Thuận,854131,,,
+Cà Mau,1286751,,,
+Cần Thơ,781655,,,
+Cao Bằng,1320324,,,
+Đà Nẵng,1463726,,,
+Đắk Lắk,1151154,,,
+Đắk Nông,1368840,,,
+Điện Biên,1892254,,,
+Đồng Nai,2028514,,,
+Đồng Tháp,1252731,,,
+Gia Lai,1860447,,,
+Hà Giang,852800,,,
+Hà Nam,1780393,,,
+Hà Nội,982487,,,
```

Comment:
The source PDF in the `.md` file has a different number for Ha Noi (8053663) on page 9. Does the page number mentioned in the `.md` file refers to households, not population. Please check.